### PR TITLE
[Draft] Runtime circle schema

### PR DESCRIPTION
**Pull request by hseok-oh**
Update tflite and circle schema

Draft for #8712

### CODE REVIEW

File: runtime/onert/frontend/base_loader/include/base_loader.h [hunk 1]
```diff
@@ -977,13 +977,12 @@ void BaseLoader<LoaderDomain>::loadBatchMatMul(const Operator *op, ir::Graph &su
...
-      // Different option name makes build fail
-      // Circle: adjoint_lhs, adjoint_rhs
-      // TFLite: adj_x, adj_y
-      // TODO Fix this issue
-      param.adj_x = op->builtin_options_as_BatchMatMulOptions()->adjoint_lhs();
-      param.adj_y = op->builtin_options_as_BatchMatMulOptions()->adjoint_rhs();
-      break;
+      // Handled on each loader: different option name
+      //  Circle: adjoint_lhs, adjoint_rhs
+      //  TFLite: adj_x, adj_y
```
Comment (chunseoklee) [inline runtime/onert/frontend/base_loader/include/base_loader.h:982]: Wow, they changed option name field .....

Comment (seanshpark) [reply runtime/onert/frontend/base_loader/include/base_loader.h:982]: Maybe we used `adjoint_lhs, adjoint_rhs` from under development code and then TFlite released with `adj_x, adj_y`

File: tests/nnfw_api/src/GenModelTests.cc [hunk 2]
```diff
@@ -308,7 +308,7 @@ TEST_F(GenModelTest, Reshape_without_shape_param)
   CircleGen::Shape new_shape_val{2, 2};
   _context = std::make_unique<GenModelTestContext>(cgen.finish());
   _context->addTestCase(uniformTCD<int32_t>({{1, 2, 3, 4}, new_shape_val}, {{1, 2, 3, 4}}));
-  _context->output_sizes(0, sizeof(i32) * 4);
+  _context->output_sizes(0, sizeof(int32_t) * 4);
```
Comment (hseok-oh) [inline tests/nnfw_api/src/GenModelTests.cc:311]: Type of `i32` (enum `circle::TensorType`) is changed to `int8_t` by `flatc` v2.0.0.

File: runtime/onert/frontend/base_loader/include/base_loader.h (hunk 1 above)
Comment (hseok-oh) [reply runtime/onert/frontend/base_loader/include/base_loader.h:992]: Right. We introduced them from `v2.2.0-rc2`, but option field names are changed on release version.

**Pull request closed without merging**